extensions/wikia/Search/classes/Result.php

```diff
@@ -357,10 +431,13 @@ public static function limitTextLength( $textAsString, $wordLimit ) {
 			$textAsString = implode( ' ', array_slice( $wordsExploded, 0, $wordLimit ) );
 			if ( count( $wordsExploded ) > $wordLimit ) {
 				$textAsString = static::fixSnippeting( $textAsString, true );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$textAsString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)